### ISSUE TITLE
Random object file (.o) appears in the folder

### ISSUE THREAD

**Issue by wil93** [hebaishi/linter-gcc#97]
Whenever I save (and the linter gets triggered) an object file `.o` appears in the folder. It doesn't even appear in the same folder as the file I'm editing, it randomly appears 2 folders above it.

I have GCC version 6.2.1.

See the issue here: http://webmshare.com/play/WJQnd

**Comment by Madh93** [comment]
I have the same problem.

**Comment by wil93** [comment]
I am on latest release (0.7.1) and the issue persists.

**Comment by hebaishi** [comment]
This is how linter-gcc works - by compiling your source file. To remove the generated object file file, simply add ```-o /dev/null``` to your command line options for Linux and ```-o NUL``` for Windows

**Comment by wil93** [comment]
Shouldn't that be the default?

Il mar 14 feb 2017, 07:59 Husam Hebaishi <notifications@github.com> ha
scritto:

**Comment by wil93** [comment]
By the way, adding `-fsyntax-only` solves the issue for me (with g++ 6.3.1).

Any change this flag can be added by default?
